scripts/changelog-check

```diff
@@ -63,6 +63,21 @@ def commit_messages_contain_skip_changelog?(base_branch, source_branch)
   log.include?(SKIP_CHANGELOG_MESSAGE)
 end
 
+def generate_invalid_changes(base_branch, source_branch)
+  log = get_git_log(base_branch, source_branch)
+  log.map do |commit|
+    change = build_changelog_from_commit(commit)
+    next if commit.title.include?(SKIP_CHANGELOG_MESSAGE)
+    next if commit.commit_messages.any? { |message| message.include?(SKIP_CHANGELOG_MESSAGE) }
+
+    if change
+      nil
+    else
+      commit.title
+    end
+  end.compact
```

Comment:
The last step seemed backwards to me so maybe we separate this into a `.select.map` ?

```suggestion
  log.reject do |commit|
    commit.title.include?(SKIP_CHANGELOG_MESSAGE) ||
      commit.commit_messages.any? { |message| message.include?(SKIP_CHANGELOG_MESSAGE) } ||
      build_changelog_from_commit(commit)
    end.map(&:title)
  end
```